core/domain/exp_domain.py

```diff
@@ -594,6 +751,12 @@ class TriggerInstance(object):
     or a certain amount of time having elapsed.
     """
     def __init__(self, trigger_type, customization_args):
+        """Initializes a TriggerInstance domain object.
+
+        Args:
+            trigger_type: str. The type of trigger.
```

Comment:
The only trigger types at the moment are NthResubmission and ClickButton -- see extensions/triggers/trigger_classes.py.

That said, let's just leave this as-is for now -- as it turns out, triggers are in the process of being removed from the codebase, due to the recent work on converting fallbacks to hints.